Job/DoctrineJobRepository.php

```diff
@@ -65,9 +71,20 @@ public function getJobManager()
     /**
      * {@inheritdoc}
      */
+    public function getJobExecution(JobInstance $jobInstance)
```

Comment:
Yes, I need to set user to jobexecution here: https://github.com/akeneo/BatchBundle/pull/61/files#diff-bb560c29dc31bd552216ce92c0673434R143